code/controllers/subsystem/non_firing/SSmapping.dm

```diff
@@ -118,6 +118,40 @@ SUBSYSTEM_DEF(mapping)
 	else
 		world.name = station_name()
 
+	// hehe
+	// try catch here so if it fails it doesn't actually kill the server
+	try
```

Comment:
can you make this a proc on `/datum/controller/subsystem/mapping` and just call it instead of having an try catch with empty catch

Reply:
eh, its good enough for an AFD PR, it works.